src/components/ButtonPanel.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Button from './Button';
+import '../styles/styles.css';
```

Comment:
You should have the same result as the project spec given by Microverse, screenshots below;
You already passed the needed props to the `Button` component, but we also need to pass the props in here too, 

![image](https://user-images.githubusercontent.com/31897434/80662583-88e3bb00-8a89-11ea-8f2c-82f8359beddd.png)

![image](https://user-images.githubusercontent.com/31897434/80662802-1a532d00-8a8a-11ea-8d58-96710a9c6ba8.png)

Example below: _let's try this_
Because by default button are orange so we need to alter them:
`<Button name="7" color ='grey'/>`
Our button with `0` should take the props `mid` right?
`<Button name="0" mid />`